testing/tests/simple.rs

```diff
@@ -484,3 +484,16 @@ fn test_num_literals() {
         "[90, -90, 90, 2, 56, 240, 10.5, 10.5, 100000000000, 105000000000]",
     );
 }
+
+#[allow(non_snake_case)]
```

Comment:
Would be nice to add comments explaining what this is testing and also eventually link to the issue.

Reply:
Added.